recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "huggingface_hub" %}
-{% set version = "0.20.3" %}
+{% set version = "0.23.0" %}
```

```suggestion
{% set version = "0.23.1" %}
```
They just released a hotfix https://github.com/huggingface/huggingface_hub/releases/tag/v0.23.1
https://github.com/huggingface/huggingface_hub/pull/2271
Mentioned as solving bugs in datasets.